src/components/Modal.js

```diff
@@ -60,8 +61,8 @@ const Modal = React.createClass({
 
   componentDidMount () {
   /*eslint-disable */
-    if (this.props.isOpen !== null) {
-      console.warn('WARNING: The prop "isOpen" is depracated in this version of the component. Please handle Modal opening from its parent.');
+    if (_has(this.props, 'isOpen')) {
```

Comment:
I'm pretty sure `if(this.props.isOpen)` will only check to see if the prop is false, not if it exists